packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 52.2.0 - June 24, 2021
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Save fileproperties after deploy ([PR #3334](https://github.com/forcedotcom/salesforcedx-vscode/pull/3334))
```

Comment:
All of these PRs are associated with Conflict Detection, which has not been released to the customers yet. Should I replace all of these with something more generic ie. `Background enhancements to conflict detection caching`?

Reply:
@AnanyaJha - Yes, I think adding the conflict detection context and making it more general would be more useful to customers.  Wording you suggested looks ok